packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -78,6 +80,7 @@ function CustomDrawerItemList({
   ...passThroughProps
 }: CustomDrawerItemListProps) {
   const buildLink = useLinkBuilder()
+  const navigationState = useNavigationState((navState) => navState)
```

Comment:
This will cause this component to re-render on every navigation state change, which might be ok but should better be avoided.

Since this is used only in `onPress`, what do you think of using `navigation.dangerouslyGetState()` there instead?
(though maybe this won't work quite right because of nesting, and we might need to get the parent navigation).

Reply:
Thanks for catching this! It did actually work since we were already passing the parent navigation :)